functions/src/model/ciJobs.ts

```diff
@@ -138,8 +139,17 @@ export class CiJobs {
     repoVersionInfo: RepoVersionInfo,
     editorVersionInfo: EditorVersionInfo | null = null,
   ): CiJob => {
+    let status: JobStatus = 'deprecated';
+    if (
+      editorVersionInfo === null ||
+      editorVersionInfo.major >= 2019 ||
+      (editorVersionInfo.major === 2018 && editorVersionInfo.minor >= 2)
+    ) {
+      status = 'created';
+    }
```

Comment:
Would it be more readable to have a `isDeprecated` function?

Reply:
A boolean containing info about whether the status is deprecated or not would be called that.

i.e. `isDeprecated = status === 'deprecated'`.